build.sbt

```diff
@@ -68,14 +76,18 @@ lazy val anorm = project
       // New functions
       missMeth("anorm.Sql.unsafeStatement$default$2"),
       missMeth("anorm.Sql.unsafeResultSet"),
-      missMeth("anorm.Sql.unsafeStatement")),
+      missMeth("anorm.Sql.unsafeStatement"),
+      missMeth("anorm.ToStatement.contramap"),
```

Comment:
> everyone would update and release their library as soon as a new release of any dependency comes out

Or rather use `provided` scope to allow to avoid dependency hell.

Even if I would not guarantee this edge case, I can see the point, and this case better to bump to 2.6.0 finally.

The same issue with transitive deps could still happen then, but if we agree the guarantee does not apply in this case.